readme.md

```diff
@@ -167,9 +165,33 @@ You can programmatically access Favicons configuration (icon filenames, HTML, ma
 import { config } from "favicons";
 ```
 
-## Output
+Below you will find a simple working example to generate an output. Amend the `src`, `dest`, `htmlBasename` and `configuration` constants to suit your own needs.
+
+```js
+import favicons from 'favicons';
+import fs from 'fs';
```

Comment:
As soon as `favicons()` is already asynchronous, it would be cleaner to use `fs/promises` instead.